docusaurus.config.js

```diff
@@ -84,7 +86,7 @@ module.exports = {
           title: 'Playgrounds',
           items: [
             {
-              label: 'OpenHIM 5.2-latest Sandbox',
+              label: 'OpenHIM 6.0-latest Sandbox',
```

Comment:
The sandbox is still on v5.2 - are we updating this soon?

Reply:
I think once we have the newer 6.x.x changes all in the 6.0.0.alpha branch we can then deploy the new sandbox environment. These docs will only be published once we have those changes in the alpha branch, or we have officially merged into master